tests/certification/state/mysql/README.md

```diff
@@ -0,0 +1,32 @@
+# MySQL State Store certification testing
+
+This project aims to test the MySQL State Store component under various conditions. Tests are run against both MySQL 8 and MariaDB 10.
+
+## Test plan
+
+## Test for CRUD operations
+
+1. Able to create and test connection.
```

Comment:
We do not need to test these things here because it is covered as part of conformance tests, and it really irks me to have duplicate test cases in certification tests :)